docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,15 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service Has Been Forced to Roll Back to Zero (CB90067)
+
+*Background*: A service has been forced to roll back to zero. This is bad because it means the service is attempting to resynchronise the entirety of the data set.
```

Comment:
One sentence per line please (also I'd perhaps change "this is bad" to "this may be undesirable" or something similar?)

Also, may be worth explaining what a rollback is - it's a Couchbase-specific term that sysadmins may not know.

Reply:
Ah sorry my bad! I'll fix the line issue. I'll also add an explanation for what a rollback is